UI/PasswordEnterWnd.cpp

```diff
@@ -0,0 +1,100 @@
+#include "PasswordEnterWnd.h"
+
+#include <GG/GUI.h>
+
+#include "../util/i18n.h"
+#include "../network/ClientNetworking.h"
+#include "../client/human/HumanClientApp.h"
+
+namespace {
+    const GG::X WINDOW_WIDTH(400);
+    const GG::Y WINDOW_HEIGHT(535);
+}
+
+PasswordEnterWnd::PasswordEnterWnd()
+    :CUIWnd(UserString("AUTHENTICATION_WINDOW_TITLE"), GG::X(80), GG::Y(130), WINDOW_WIDTH, WINDOW_HEIGHT,
```

Comment:
`:` to previous line (with space between `)` and `:`)

Reply:
Fixed.